static/app/components/events/aiSuggestedSolution/experimentalFeatureBadge.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+
+import FeatureBadge from 'sentry/components/featureBadge';
+import {t} from 'sentry/locale';
+
+export function ExperimentalFeatureBadge() {
+  return (
+    <CapitalizesFeatureBadge
+      size="sm"
+      type="experimental"
+      title={t(
+        'This is an OpenAI generated solution that suggests a fix for this issue. Be aware that this may not be accurate.'
+      )}
+    />
+  );
+}
+
+const CapitalizesFeatureBadge = styled(FeatureBadge)`
+  text-transform: capitalize;
```

Comment:
Do we really have to do this? We don't do this anywhere else?

Reply:
I understand your concern and noticed the same, but everything uppercase seems so aggressive 🤔 in any case for the sake of consistency, I agree and think we should remove this change... are you fine with uppercasing the tag @Jesse-Box ?

Reply:
I'm fine with using existing patterns here